build/helper/doc_snippets.py

```diff
@@ -0,0 +1,34 @@
+# Different documentation snippets we add to the generated documentation
```

Comment:
this file is doc_snippets.py, and we have documentation_helper.py.
We should use same prefix.